### PR TITLE
MessageOptions#code accepts a boolean too and new typings

### LINKED ISSUE #38: MessageOptions property 'code' should be type 'string' | 'boolean'

### PR DESCRIPTION
**Pull request by SpaceEEC**
Fixes #38 - `MessageOptions#code` accepts a boolean now.

Also includes typings from the following commits:
- `Guild#nameAcronym`: https://github.com/hydrabolt/discord.js/commit/cd4a69d009e9765f2f2ca87feede7435abdda902 
- Voice `bitrate` support: https://github.com/hydrabolt/discord.js/commit/cba4cc2400038bb914fc0adf64ad80383601eade
- `Collection` count parameter: https://github.com/hydrabolt/discord.js/commit/d513c4bbb9bb845fc64dcfe250ff0f1b95c0ee2a

### CODE REVIEW

**Comment by zajrik** [comment]
Is this v12? I can't tell from the commits on mobile

**Comment by SpaceEEC** [comment]
I should have market it in the title or somehwere else.
It's for 11.2; new commits in the 11.1-dev branch.

File: index.d.ts [hunk 1]
```diff
@@ -243,13 +246,19 @@ declare module 'discord.js' {
...
 		public findKey(fn: (value: V, key: K, collection: Collection<K, V>) => boolean): K;
 		public first(): V;
+		public first(count: number): V[];
 		public firstKey(): K;
+		public firstKey(count: number): V[];
 		public keyArray(): K[];
 		public last(): V;
+		public last(count: number): V[];
 		public lastKey(): K;
+		public lastKey(count: number): V[];
```
Comment (zajrik) [inline index.d.ts]: This should return `K[]`

File: index.d.ts [hunk 2]
```diff
@@ -243,13 +246,19 @@ declare module 'discord.js' {
...
 		public map<T>(fn: (value: V, key: K, collection: Collection<K, V>) => T, thisArg?: any): T[];
 		public random(): V;
+		public random(count: number): V[];
 		public randomKey(): K;
+		public randomKey(count: number): V[];
```
Comment (zajrik) [inline index.d.ts]: This should return `K[]`

File: index.d.ts [hunk 3]
```diff
@@ -243,13 +246,19 @@ declare module 'discord.js' {
 		public findKey(prop: keyof V, value: any): K;
 		public findKey(fn: (value: V, key: K, collection: Collection<K, V>) => boolean): K;
 		public first(): V;
+		public first(count: number): V[];
 		public firstKey(): K;
+		public firstKey(count: number): V[];
```
Comment (zajrik) [inline index.d.ts]: This should return `K[]`

**Comment by zajrik** [comment]
Added some comments. Wouldn't let me do a review on my mobile client.

**Comment by SpaceEEC** [comment]
Whoops, is fixed.

**Comment by zajrik** [comment]
👌

**Pull request merged**